django/thunderstore/repository/storage.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
```

Comment:
This entire file should probably not exist (as we don't need provider-specific storages, just the stub storage). The `STORAGE_CLASS_HANDLER`  on the other hand should probably be defined somewhere else, not in the `thunderstore.repository` module. **Unless** you rename it to `REPOSITORY_STORAGE` and it's used only for the repository content, but that's not the case here.

So in short, get rid of this file but move `STORAGE_CLASS_HANDLER` to somewhere shared e.g. `thunderstore.core.storage` and also rename it to just `DEFAULT_STORAGE` or something.